.github/workflows/check_results.yml

```diff
@@ -0,0 +1,177 @@
+name: check_results
```

Comment:
Can you add a few comment lines at the top of this pipeline to summarize what it does, results/artifacts that it depends on and that produces?